### PR TITLE
docs(aio): add my details as a contributor

### PR DESCRIPTION
**Pull request by gsans**

### CODE REVIEW

File: aio/content/marketing/contributors.json [hunk 1]
```diff
@@ -573,5 +573,14 @@
    "website": "http://www.methotic.com",
    "bio": "Thierry is a senior consultant and trainer, specialized on Angular, and a Google Developer Expert.",
    "group": "GDE"
+ },
+
+"gerardsans": {
+   "name": "Gerard Sans",
+   "picture": "gerardsans.jpg",
```
Comment (gkalpak) [inline aio/content/marketing/contributors.json]: [Where][1]'s the picture? :smiiley:

[1]: https://github.com/gsans/angular/tree/a2c4d07d23836746d2de09cb293208121c197493/aio/content/images/bios

Comment (gsans) [inline aio/content/marketing/contributors.json]: Yep. Here https://drive.google.com/drive/folders/0B_z6OFiRbgQiQmVQZkR0dkJRY0E

Comment (gkalpak) [inline aio/content/marketing/contributors.json]: AFAIK, you need to put it [here][1] (as part of this PR) :smiley:

[1]: https://github.com/gsans/angular/tree/a2c4d07d23836746d2de09cb293208121c197493/aio/content/images/bios

Comment (gsans) [inline aio/content/marketing/contributors.json]: Sure thing. Done this at https://github.com/angular/angular/pull/17683

Comment (gkalpak) [inline aio/content/marketing/contributors.json]: This is #17683 (and I still don't see an image file) 😕

**Comment by IgorMinar** [comment]
@gsans please add your photo to this PR

**Comment by gsans** [comment]
Sorry about that. Is it ok now?

**Pull request closed without merging**